src/qml/BrowserView.qml

```diff
@@ -95,4 +102,66 @@ Rectangle {
 
         onClicked: view.selectionAccepted()
     }
+
+    Popup {
+        id: fullPopup
+        modal: true
+
+        property alias name: nameText.text
+        property alias value: valueText.text
+
+        implicitWidth: popupFlickable.width
+        implicitHeight: popupFlickable.height
+        padding: 0
+
+        clip: true
+
+        anchors.centerIn: browser
+
+        background: Rectangle {
+            radius: 3
+            opacity: 0.8
+            color: Style.contextMenu.backgroundSelected
```

Comment:
Each view has its own style element. It would therefore be better to extend the ThemeBrowserView element with popupBackground and popupTextColor.